src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -409,6 +409,23 @@ def reverse_resolve(self, use_case_id: UseCaseID, org_id: int, id: int) -> Optio
         string: str = obj.string
         return string
 
+    def bulk_reverse_resolve(
+        self, use_case_id: UseCaseID, org_id: int, ids: Collection[int]
+    ) -> Mapping[int, str]:
+        ret_val: Dict[int, str] = {}
+        metric_path_key = METRIC_PATH_MAPPING[use_case_id]
+        table = self._get_table_from_metric_path_key(metric_path_key)
+        try:
+            strings = table.objects.get_many_from_cache(ids)
```

Comment:
I would assume the values do not change, thus we shouldn't have problems with cache inconsistency, right?

Reply:
Yes, we are counting on them not changing since they are fixed in Snuba. 
We expect the mapping not to change at least for the life of the entry in Snuba otherwise we couldn't map back to the original string.